fedimint-atomic-broadcast/src/broadcast.rs

```diff
@@ -92,7 +94,7 @@ impl AtomicBroadcast {
         if let Some(signed_block) = db::load_block(&self.db, index).await {
             tracing::info!("Loaded block with index {}", index);
 
-            tokio::spawn(async move {
+            spawn("atomic run session", async move {
```

Comment:
Might it be useful to distinguish the two scenarios? Maybe call this one `atomic run session db`